package.json

```diff
@@ -15,6 +15,7 @@
   "exports": {
     ".": {
       "types": "./src/index.d.ts",
+      "module-sync": "./src/index.js",
```

Comment:
The actual `exports` https://github.com/prettier/prettier/blob/eed53fc29a36a0d045a3768e1aba2f61cb5b593a/scripts/build/build-package-json.js#L42